### PR TITLE
Added kwargs to Excel UI process beads table.

### PR DESCRIPTION
**Pull request by JS3xton**
See #222.

### CODE REVIEW

**Comment by JS3xton** [comment]
As of e1111cf, all unit tests pass:

```
> python -m unittest discover
........................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 280 tests in 5.213s

OK
```

Tested manually and was able to turn off subpopulation selection step:
```python
FlowCal.excel_ui.process_beads_table(beads_table, instrument_table, plot=True)
```
yielded:
populations_fl3_b1

```python
FlowCal.excel_ui.process_beads_table(beads_table, instrument_table, plot=True, get_transform_fxn_kwargs={'selection_fxn':None})
```
yielded:
populations_fl3_b1

(FL3 channel; top peak was being explicitly excluded in the input XLSX sheet)

**Comment by castillohair** [comment]
Just to confirm, there are two peaks to the right being excluded in the second plot of your example. Are you explicitly excluding both?

**Comment by JS3xton** [comment]
Yep. This should be the `Beads` tab of the XLSX sheet for that plot:

excel_ui

File: FlowCal/excel_ui.py [hunk 1]
```diff
@@ -212,6 +212,7 @@ def write_workbook(filename, table_list, column_width=None):
 
 def process_beads_table(beads_table,
                         instruments_table,
+                        get_transform_fxn_kwargs={},
```
Comment (castillohair) [inline FlowCal/excel_ui.py]: I think this should be the last parameter. This is such an edge case that I don't think changing the order of parameters is appropriate.

**Review by castillohair — CHANGES_REQUESTED** [review]
The order of parameters is the only complain I have.

**Review by JS3xton — COMMENTED** [review]
updated as per your request

**Pull request merged**